README.md

```diff
@@ -24,7 +31,7 @@ environment variables, or with a link to a configuration guide.
 
 | Environment variable | Default | Description
 | -------------------- | ------- | -----------
-| BIND_ADDR            | :8080   | The host and port to bind to
+| BIND_ADDR           | :23000   | The port to bind to
```

Comment:
Earlier you mention port 5005 and the URL in the instructions is for `localhost:2300` - missing an extra 0?